src/mqt/yaqs/core/data_structures/noise_model.py

```
@@ -71,125 +248,180 @@ def __init__(
+        else:
+            jump_dict["sites"] = sites
+
+        if "matrix" in jump_dict:
+            jump_dict["matrix"] = _as_square_matrix(jump_dict["matrix"], "Scheduled jump matrix")
+        else:
+            jump_dict["matrix"] = NoiseModel.get_operator(jump_dict["name"])
+        return jump_dict
 
-        self.processes = filled_processes
+    @staticmethod
+    def _normalize_process(original: object) -> dict[str, Any]:
```

Ruff (0.16.1)

[warning] Too many branches (21 > 12)

(PLR0912)